templates/bridgeserver2.yaml

```diff
@@ -631,7 +631,7 @@ Resources:
         - - /aws/elasticbeanstalk
           - !Ref 'AWS::StackName'
           - var/log/tomcat8/catalina.out
-      FilterPattern: 'WARN - "No content length specified for stream data"'
+      FilterPattern: 'WARN'
```

Comment:
Does this mean we're now filtering out all WARN statements, not just the warn statements based on this content length issue?

Reply:
The - means NOT. In this case, we have an alarm on WARN but not "No content lenght specified for stream data". Now, the alarm is on all WARN.

In short, it's the opposite. Before, we were excluding all content length warnings from the alarm. Now, we have no exclusions on the warning alarm.